docs/user_guide/index.md

```diff
@@ -12,106 +12,10 @@ self
-
-   - Methods to find incoming and outgoing edges for nodes.
-   - Can determine edge endpoints and find edges connecting specific nodes.
-   - Supports both directed and undirected neighbor queries.
-
-7. **Clearing and Counting**:
-
-   - Can clear all data in the record.
-   - Provides methods to count nodes, edges, and groups.
-
-8. **Properties and Accessors**:
-   - Properties to get node, edge, and group lists.
-   - Provides indexers for node and edge attributes, allowing flexible querying and manipulation.
-
-Overall, the `MedRecord` class is a robust tool for handling complex medical data with a flexible and efficient approach to manage nodes (patients, conditions, etc.) and edges (relationships, treatments, etc.) in a graph-based structure.
-
-:::
-
-:::{admonition} `TreatmentEffect`
-:class: seealso, dropdown
-
-The {py:class}`TreatmentEffect <medmodels.treatment_effect.treatment_effect.TreatmentEffect>` class is designed for analyzing treatment effects within medical records. The class has the following capabilities:
-
-1. **Initialization and Configuration**:
-
-   - Initializes with groups of treatment and outcome nodes.
-   - Configurable parameters include patient groups, time attributes, washout periods, grace periods, follow-up periods, and criteria for filtering control groups.
-
-2. **Group Identification and Filtering**:
-
-   - Identifies patients who underwent treatment and experienced outcomes.
-   - Finds control groups with similar criteria but without undergoing the treatment.
-   - Applies customizable criteria filtering and time constraints between treatment and outcome.
-
-3. **Matching and Comparison**:
-
-   - Supports optional matching of control groups to treatment groups using specified matching methods and covariates.
-   - Parameters for matching include essential covariates, one-hot covariates, matching models, number of neighbors, and hyperparameters.
-
-4. **Outcome Analysis**:
-
-   - Determines patients who had the outcome after the treatment, and optionally before the treatment if configured.
-   - Applies washout periods to exclude patients who experienced outcomes during specified times.
-
-5. **Control Group Handling**:
-
-   - Identifies control patients and applies filters to ensure appropriate comparison groups.
-   - Distinguishes between control patients who experienced the outcome and those who did not.
-
-6. **Estimation and Reporting**:
-
-   - Provides properties to create `Estimate` and `Report` objects for detailed analysis and reporting of treatment effects.
-
-7. **Support for Temporal Analysis**:
-   - Utilizes functions like `find_node_in_time_window` for temporal analysis of treatment and outcome relationships.
-
-Overall, the `TreatmentEffect` class offers a robust framework for analyzing the impact of treatments in medical records, with flexible configurations for group identification, filtering, matching, and outcome analysis.
-
-:::
+01_install_medmodels
+02_medrecord_intro
```

Comment:
Intro, Schema and Query Engine should be subchapters of a MedRecord chapter. Looks a bit weird right now imo
<img width="339" alt="image" src="https://github.com/user-attachments/assets/4275a003-9c79-4bc4-9351-41353b27003c">

Reply:
subchapters as in:

```
Contents:
├─ Install MedModels Locally
├─ MedRecord
│  ├ Intro (either here or in the MedRecord chapter)
│  ├ Schema
│  └ Query Engine
├─ Treatment Effect
│  ├ Intro
│  ├ ...
│  └ ...
└─ Data Comparer
```

Reply:
Thats a very good idea!